transforms/universal/pdf2md/python/README.md

```diff
@@ -0,0 +1,60 @@
+# Ingest PDF to Parquet
+
+This tranforms iterate through PDF files or zip of PDF files and generates parquet files
+containing the converted document in Markdown format.
+
+The PDF conversion is using the [Docling package](https://github.com/DS4SD/docling).
+
+
+## Output format
+
+The output format will contain all the columns of the metadata CSV file,
+with the addition of the following columns
+
+```jsonc
+{
+    // all the fields of the input CSV + ...
+
+    "source_filename": "string",  // the basename of the source archive or file
+    "filename": "string",         // the basename of the PDF file
+    "contents": "string",         // the content of the PDF
+    "document_id": "string",      // the document id, corresponding to the hash of the content 
+    "ext": "string",              // the detected file extension
+    "hash": "string",             // the hash of the `contents` column
```

Comment:
so doc_id == hash. Why have both?

Reply:
now we used `uuid4` as in the other examples

Reply:
so, at least change the comment to define which is which

Reply:
yes, this was done in commit https://github.com/IBM/data-prep-kit/commit/1a8f0dbc13256c3fa76ec465fb16270aa0930c8a (5 days ago)